pkg/bundle/bundle.go

```diff
@@ -116,7 +116,13 @@ type Action struct {
 	// Modifies indicates whether this action modifies the release.
 	//
 	// If it is possible that an action modify a release, this must be set to true.
-	Modifies bool
+	Modifies bool `json:"modifies,omitempty"`
```

Comment:
Any specific reason why this doesn't also include the `mapstructure:"modifies"` tag?

Same for `Stateless` and `Description`.

Reply:
My bad, will fix that